src/screens/onboarding/views/Permissions.tsx

```diff
@@ -1,34 +1,31 @@
 import React from 'react';
-import {ScrollView} from 'react-native';
 import {Box, Text} from 'components';
 import {useI18n} from '@shopify/react-i18n';
 
 export const Permissions = () => {
   const [i18n] = useI18n();
   return (
-    <ScrollView showsVerticalScrollIndicator={false}>
```

Comment:
I think we need to keep ScrollView here or on devices with larger text sizing it will all be cutoff.

Reply:
👌 I will update.